hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1200,7 +1200,8 @@ message DeletedKeys {
 
 message PurgeKeysRequest {
     repeated DeletedKeys deletedKeys = 1;
-    optional string fromSnapshot = 2;
+    // if not null, will purge keys in a snapshot DB instead of active DB
```

Comment:
nit: sorry for the back and forth. I guess `null` was not a concept in protobuf.
```suggestion
    // if set, will purge keys in a snapshot DB instead of active DB
```